pkg/console/subresource/consoleserver/types.go

```diff
@@ -54,9 +58,10 @@ type ServingInfo struct {
 
 // ClusterInfo holds information the about the cluster such as master public URL and console public URL.
 type ClusterInfo struct {
-	ConsoleBaseAddress string `yaml:"consoleBaseAddress,omitempty"`
-	ConsoleBasePath    string `yaml:"consoleBasePath,omitempty"`
-	MasterPublicURL    string `yaml:"masterPublicURL,omitempty"`
+	ConsoleBaseAddress  string                `yaml:"consoleBaseAddress,omitempty"`
+	ConsoleBasePath     string                `yaml:"consoleBasePath,omitempty"`
+	MasterPublicURL     string                `yaml:"masterPublicURL,omitempty"`
+	ControlPlaneToplogy configv1.TopologyMode `yaml:"controlPlaneTopology,omitempty"`
```

Comment:
@jhadvig - there is a spelling mistake here that does not effect things, but should probably be fixed:  `Topology`